DEVELOPMENT.md

```diff
@@ -16,15 +16,26 @@
  4. `make sign-off` to make sure the DCO will pass.
  5. `nox --session build_package` (it builds `bundled/libs`)
 
-
- # Release
- `make build` and manually upload it to VSCode MarketPlace and Open VSX Registry.
-
 `make dev-install` installs a few different things:
 - `npm install` for the VSCode frontend
 - `pip isntall -r dev-requirements.txt` for the Python backend
 - `nox --sesssion build_pakcage` for building `bundled/libs` locally
 
+ # Release
+ Kedro VSCode is released in two places:
+ 1. [VSCode Marketplace](https://marketplace.visualstudio.com/manage/publishers/kedro/extensions/kedro)
+ 2. [Open VSX Registry](https://open-vsx.org/extension/kedro/Kedro)
+
+ The release processes are as follow:
+ 1. Bump version number
+ 2. Check `READEME.md` and `CHANGELOG.md` to make sure it reflects the latest changes.
+ 3. Run `make build` to generate the `.vsix` artifact
+ 4. Manually test if the artifact works (no automated tests yet)
+ 5. Manually upload to VSCode Marketplace and VSX Registry (Drag and Drop)
+
+To upload a release, you will need to be added as a contributor or owner for both VSCode Marketplace and VSX Registry.
```

Comment:
Hope we can automate this soon https://github.com/kedro-org/vscode-kedro/issues/128